spark/dl/src/main/scala/com/intel/analytics/bigdl/example/keras/LeNet.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.example.keras
+
+import com.intel.analytics.bigdl.Module
+import com.intel.analytics.bigdl.nn.{Sequential}
+import com.intel.analytics.bigdl.nn.keras._
+
+object LeNet {
+  def apply(): Module[Float] = {
+    val model = Sequential[Float]()
+    model.add(Reshape(Array(1, 28, 28), inputShape = Array(28, 28, 1)))
+    model.add(Convolution2D(32, 3, 3))
```

Comment:
model.add(Convolution2D(32, 3, 3), activation="relu") ?

Reply:
should be `model.add(Convolution2D(32, 3, 3, activation="relu"))`...
Currently we don't support this lol.
Only support pass in objects as activation like `model.add(Convolution2D(32, 3, 3, activation=ReLU()))`
Do we need to support string input of activation? In this case, we need a mapping utils.

Reply:
BTW, such model structure
```
model.add(Convolution2D..
model.add(Activation("relu"))
```
is from the keras example and I just follow this to create our example. Though putting `relu` into Convolution2D is has the same effect.

Reply:
model.add(Convolution2D(32, 3, 3, activation=ReLU())) is OK.

Reply:
Then I'll just keep the current implementation.
Whether to choose `model.add(Convolution2D(32, 3, 3, activation=ReLU()))` or
```
model.add(Convolution2D..
model.add(Activation("relu"))
```
just depends on personal preference haha